string support_leg
float32 duration
